x/pairing/README.md

```diff
@@ -95,6 +97,12 @@ A provider can unstake and retrieve their coins. When a provider unstakes, they
 
 Freeze Mode enables the Provider to temporarily suspend their node's operation during maintenance to avoid bad Quality of Service (QoS). Freeze/Unfreeze is applied on the next Epoch. The Provider can initiate multiple Freeze actions with one command.
 
+#### Jail
```

Comment:
Please update the "Unresponsiveness" section of the README (in this file) with the new jailing mechanism.
Also, please use Notion AI to improve writing.

Reply:
done

Reply:
https://github.com/lavanet/lava/pull/1443/commits/db61538dcf2d3cb2323588e57fc499dc51bc3b7f

Reply:
Stuff to fix:
1. The first **two** jailings are one hour, not three
2. Emphasize that jailing = freezing but jail can't be unfrozen before the jailing period. Soft jail is 1H + automatic unfreeze and Hard jail is 24H + manual unfreeze.

Reply:
https://github.com/lavanet/lava/pull/1443/commits/2c93939072c2bf1389ddceceb12a8df26edecd4e